### DISCUSSION TITLE
Force RTL for Arabic, otherwise Arabic display is messed up

### DISCUSSION THREAD

**Discussion by abosaad11** [louislam/dockge#242] [General]
When using Arabic and English words in the same text field, Right to Left (RTL) should be forced somehow. I just took a look at the Arabic display and there's a problem. I don't know how to fix it, but my suggestion is to force RTL when Arabic (or any other RTL languages) are used.

![dockge](https://github.com/louislam/dockge/assets/100481472/901ce3f8-93ca-46ec-94af-dbb557ab4c85)

In the image, `Dockge` should move with the arrow when RTL is used.

**Comment by louislam** [comment]
Arabic did not marked as RTL.

It should be fixed: https://github.com/louislam/dockge/commit/4e899dcf212cf28a2174cc2be5b0dd6b3919559d

**Comment by abosaad11** [comment]
Great thank you.
